src/status_im/ui/components/bottom_bar/core.cljs

```diff
@@ -66,25 +67,84 @@
      [react/view
       [content active? count]]]))
 
+(def height 52)
+
+:message
+:dapp
+:wallet
+:profile
+(defn new-tab [{:keys [icon label active?]}]
+  [react/touchable-highlight
+   {:style    {:flex   1
+               :height height}
+    :on-press #(js/alert icon)}
+   [react/view
+    {:style
+     {:flex            1
+      :height          height
+      :align-items     :center
+      :justify-content :space-between
+      :padding-top     6
+      :padding         4}}
+    [vector-icons/icon icon
+     {:color  (if active? colors/blue colors/gray)
+      :height 24
+      :width  24}]
+    [react/view {:style {:align-self      :stretch
+                         :height          14
+                         :align-items     :center
+                         :justify-content :center}}
+     [react/text {:style {:color     (if active? colors/blue colors/gray)
+                          :font-size 11}}
+      label]]]])
+
 (defn tabs [current-view-id]
-  [react/view {:style tabs.styles/tabs-container}
-   (for [{:keys [content view-id accessibility-label count-subscription]} tabs-list]
-     ^{:key view-id} [tab view-id content (= view-id current-view-id) accessibility-label count-subscription])])
+  [react/view
+   {:style {:height         height
+            :align-self     :stretch
+            :shadow-radius  4
+            :shadow-offset  {:width 0 :height -5}
+            :shadow-opacity 0.3
+            :shadow-color   "rgba(0, 9, 26, 0.12)"}}
+   [react/view {:style {:height         height
+                        :align-self     :stretch
+                        :padding-left    8
+                        :padding-right   8
+                        :flex-direction :row}}
+    [new-tab {:icon    :main-icons/message
+              :label   "Chats"
+              :active? (= current-view-id :chat-stack)}]
+    [new-tab {:icon    :main-icons/dapp
+              :label   "DApps"
+              :active? (= current-view-id :dapp-stack)}]
+    [new-tab {:icon    :main-icons/wallet
+              :label   "Wallet"
+              :active? (= current-view-id :wallet-stack)}]
+    [new-tab {:icon    :main-icons/profile
```

Comment:
hey @rasom could we correct the icon for profile as part of this PR, it should be using the `user_profile` and not `profile` 🙏

Reply:
sure